base/applications/network/ipconfig/ipconfig.c

```diff
@@ -25,6 +25,7 @@
 #include <ndk/rtlfuncs.h>
 #include <inaddr.h>
 #include <windns.h>
+#include <winnls.h>
```

Comment:
Please move this up, between winbase and winuser.

Reply:
committed